### ISSUE TITLE
[BUG] Axe sometimes shows fewer failures than AccessibilityInsights

### ISSUE THREAD

**Issue by donnie-msft** [microsoft/axe-windows#628]
Please check whether the bug has [already been filed](https://github.com/Microsoft/axe-windows/issues).

## Describe the bug

I'm testing Axe, and when I compare the results to running AccessibilityInsights directly, I find Axe is missing some failures.
I'm using NuGet package Axe.Windows 1.1.0, which is the same version that my AccessibilityInsights was built with.

Is there any way I can determine if this is an Axe bug? How reliable should Axe be compared to AccessibilityInsights?

**To Reproduce**

For the first several runs, it missed 1 of the 2 failures for **"An element's IsOffScreen property must be false when its clickable point is on screen. "**.

AccessibilityInsights reports 2 errors:
Section 508 502.3.1: (devenv/list item 'Swashbuckle.AspNetCore.Swagger'
Section 508 502.3.1: (devenv/list item 'Swashbuckle.AspNetCore.SwaggerGen' **<-- this one was always missing from Axe results.**

Later, I ended up seeing 0 of the 2 failures above in Axe. Nothing changed. I left the application running, but just reran my Axe scan many times.

`ErrorCount` here is 16 or 17:

```
            // Create config to specifically target a process
            var myConfigBuilder = Config.Builder.ForProcessId(38552);

            var myConfig = myConfigBuilder.Build();
            // Create scanner using myConfig
            var scanner = ScannerFactory.CreateScanner(myConfig);

            ScanResults scanResults;
            try
            {
                scanResults = scanner.Scan();
                Console.WriteLine("Number of errors found in scan: " + scanResults.ErrorCount);
```

## Expected behavior

The same 18 errors in Axe as AccessibilityIinsights.
![image](https://user-images.githubusercontent.com/49205731/129988345-0ace2c7e-a55a-435d-ad17-f30f5b2765e8.png)

## Actual behavior

Only 17 errors in Axe. Later, only 16 errors. AccessibilityInsights always showed 18 errors.

## Desktop (please complete the following information):

 - OS: [e.g. Windows 10 1809] (Get the version by running `winver` from the command line) Version	10.0.19043 Build 19043

 - Accessibility Insights for Windows Version: 1.1.1663.1 (Axe.Windows 1.1.0)
 - Target Application: [e.g. Chrome, Wildlife Manager] WPF (NuGet Client in Visual Studio 2022)
 - Target Application Version: [e.g. 22]

## Additional context

Priority requested -

[TestResults 08-18-21.a11ytest.zip](https://github.com/microsoft/axe-windows/files/7010861/TestResults.08-18-21.a11ytest.zip)

**Comment by DaveTryon** [comment]
@donnie-msft, as we discussed on the phone, Axe.Windows is the core of AIWin, and the expectation is that they will always report the same deterministic results against the same input data. In this case, it seems like UIA is reporting different data between the different runs, and as neither you nor I were able to repro this behavior today,  it feels like a transient issue in UIA when you were running the tests.

We'll discuss this in our next triage, but without the ability to reproduce it, there's not very much that we can do to fix this. Thanks!

**Comment by DaveTryon** [comment]
@donnie-msft, as discussed, without a consistent repro there's really nothing that we can do here. Sending it back to you to please close out unless you can find a consistent repro.
